### PR TITLE
Adding steps for facebook bot integration

### PR DESCRIPTION
**Pull request by RandiveSushant**
so that these steps can be used by everyone while trying to instantiate facebook bot integration

### CODE REVIEW

File: docs/other/facebook.md [hunk 1]
```diff
@@ -2,19 +2,219 @@
...
+
+- Click on **Add Callback URL**.
+
+- Enter the following URL into the **Callback URL** field:
+
+  - > Callback URL = (Environment's base URL) + (callback_url entered in the Dialog Builder)
+
+  - _Staging_ - `https://staging-messenger.haptikapi.com/<callback_url entered in Dialog Builder>`. Thus, following our example, our complete Callback URL would be `https://staging-messenger.haptikapi.com/fb/mydemobusiness`.
+
+  - _Production_ - `https://fb-messenger.haptikapi.com/<callback_url entered in Dialog Builder>`. Thus, following our example, our complete Callback URL would be `https://fb-messenger.haptikapi.com/fb/mydemobusiness`.
```
Comment (bhaveshpatelh) [inline docs/other/facebook.md]: @RandiveSushant check with @ranvijayj If we should add production url on prod ?

Comment (RandiveSushant) [reply docs/other/facebook.md]: Okay, but this is the URL for prod message delivery.

File: docs/other/facebook.md [hunk 2]
```diff
@@ -2,19 +2,219 @@
...
+  - **Add or Remove Pages** if the page has already been created.
+
+  - **Create New Page** to create the page if you have not created it yet.
+
+
+- Now select the page which you would like to add to the app and click **continue**.
+
+- After the page has been successfully added to the app, click **Generate Token** to create the _access token_.
+
+  > PLease note:
```
Comment (medhaanand) [inline docs/other/facebook.md:80]: "Please" make the L small

File: docs/other/facebook.md [hunk 3]
```diff
@@ -2,19 +2,219 @@
...
+  >- Ask the client to **save** this _access token_ and **not to re-generate the token** as generating a new token will immediately invalidate the previously generated _access token_.<br><br>
+  >- If your app is in _development mode_, you can still generate a token but will only be able to access those Facebook users who either manage the app or the Facebook page
+
+
+
+## Step 4: Setting up the _Dialog Builder_:
```
Comment (medhaanand) [inline docs/other/facebook.md:86]: Why is it Dialog Builder? Why not Bot Builder?

File: docs/other/facebook.md [hunk 4]
```diff
@@ -2,19 +2,219 @@
...
+
+- Get the App Secret:
+  - `Facebook for devlopers -> (our app) -> settings -> basic`
```
Comment (medhaanand) [inline docs/other/facebook.md:89]: developers spelling incorrect

File: docs/other/facebook.md [hunk 5]
```diff
@@ -2,19 +2,219 @@
...
+
+  - Find the field titled **App Secret** and click on **Show**
+
+  - Copy the value in this field and paste it in a note or someplace safe and keep it handy since we would need it while going through the next steps.
+
+- Now go to `Dialog Builder -> Business Manager -> _(the business in question)_ -> Partner Specific -> Platform Deployments -> Facebook`.
```
Comment (medhaanand) [inline docs/other/facebook.md:95]: Change Dialog Builder to Bot Builder

File: docs/other/facebook.md (hunk 1 above)
Comment (RandiveSushant) [reply docs/other/facebook.md:122]: @bhaveshpatelh  you were right about the URLs. I would be removing them.

File: docs/other/facebook.md (hunk 3 above)
Comment (RandiveSushant) [reply docs/other/facebook.md]: done

File: docs/other/facebook.md (hunk 5 above)
Comment (RandiveSushant) [reply docs/other/facebook.md]: done

File: docs/other/facebook.md (hunk 2 above)
Comment (RandiveSushant) [reply docs/other/facebook.md]: Done

File: docs/other/facebook.md (hunk 4 above)
Comment (RandiveSushant) [reply docs/other/facebook.md]: done

**Pull request merged**